tests/docker/BUILD

```diff
@@ -176,3 +176,18 @@ file_test(
     content = "sha256:5126c97392b596711e00f5af71675581978475cf1490d12a55d4b321ceeee392",
     file = ":null_cmd_and_entrypoint_none.digest",
 )
+
+container_image(
+    name = "set_env_csv",
+    base = "@distroless_fixed_id//image",
+    cmd = ["foo"],
+)
+
+container_test(
+    name = "set_env_csv_test",
+    configs = ["//tests/docker/configs:set_env_csv.yaml"],
+    env = {
```

Comment:
container test does not allow env attr, it only receives the configs to validate with

Reply:
(the env should be set in the container_image above)